<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerConfigDlgClass</class>
 <widget class="QDialog" name="TimerConfigDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAdvanced">
           <property name="text">
            <string>Advanced &gt;</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Simple</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Resolution</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinResolution"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Frequency</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spinFrequency"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Actual Frequency</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lblActualFrequency">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupAdvanced">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Advanced</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Clock</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblClockFrequency">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Bits</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblBits">
            <property name="text">
             <string>16</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cboWaveformMode"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Prescale</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cboPrescale"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Counter Top</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinCounterTop"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Output Frequency</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="lblOutputFrequency">
            <property name="text">
             <string>12.1</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Data Bits</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="lblDataBits">
            <property name="text">
             <string>12.1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cboWaveformMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>ComputeTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboPrescale</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>ComputeTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinCounterTop</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>ComputeTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinResolution</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>AutoComputeTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinFrequency</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>AutoComputeTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdvanced</sender>
   <signal>toggled(bool)</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>onShowAdvanced()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>TimerConfigDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ComputeTimer()</slot>
  <slot>AutoComputeTimer()</slot>
  <slot>onShowAdvanced()</slot>
 </slots>
</ui>
